uikit/src/main/java/com/midtrans/sdk/uikit/views/banktransfer/status/BankTransferStatusPresenter.java

```diff
@@ -111,11 +128,14 @@ public String getMandiriBillExpiration() {
     }
 
     public String getBankCode() {
-        String bankCode = LABEL_BANK_CODE_BNI;
-
-        MerchantPreferences preferences = getMidtransSDK().getMerchantData().getPreference();
-        if (preferences != null && !TextUtils.isEmpty(preferences.getOtherVaProcessor())
-                && preferences.getOtherVaProcessor().equals(UiKitConstants.OTHER_VA_PROCESSOR_PERMATA)) {
+        String bankCode;
+        String bank = transactionResponse.getAccountNumbers().get(0).getBank();
+
+        if (bank.equals(BRI)) {
```

Comment:
I think using map is more appropriate

Reply:
Could you elaborate more about this one.

Reply:
you can read https://docs.oracle.com/javase/8/docs/api/java/util/Map.html as reference.